plugins/backstage-plugin-coder/package.json

```diff
@@ -46,7 +46,8 @@
     "valibot": "^0.28.1"
   },
   "peerDependencies": {
-    "react": "^16.13.1 || ^17.0.0 || ^18.0.0"
+    "react": "^16.13.1 || ^17.0.0 || ^18.0.0",
+    "react-dom": "^16.13.1 || ^17.0.0 || ^18.0.0"
```

Comment:
Needed to bring in React DOM as a peer dependency so I could start using React portals